docs/NOTES.md

```diff
@@ -124,3 +124,20 @@ for PhoenixGo:
 - the requirement to disable pondering, you need to set `enable_background_search`
 to `0` in config file.
 - show pv in stderr with `--logtostderr` and `--v=1` in command-line options.
+
+## I
+
+Rejectnewtime options requires to specify
+
+- a number of days to add, starting from which we will reject games
+- time of the day in hour:minutes local time format starting from which we will reject games,
+for example `21:15` for 9PM15, `02:45` for 2AM45
+
+Times are inputed in local times, not GMT time, so if it's 1PM30 at your place (GMT+2)
+put `13:30`, not `11:30`.
+
+Example of use:
+
+- `--rejectnewtime 0:21:30` will reject new games starting from today (0) at 09PM30 (21:30)
+- `--rejectnewtime 1:02:15` will reject new games starting from tomorrow (+1) at 02:15 AM
+- `--rejectnewtime 3:12:59` will reject new games starting from 3 days from now on (ex Monday -> Thrusday) (+3) at 12:59 AM
```

Comment:
thursday

Reply:
this does not appear now on the refactored version, fixed too :+1: